user-guide/Advanced_Functionality/Databases/Configuring_dedicated_clustered_storage/Indexing_database/OpenSearch_Database/Installing_OpenSearch_database.md

```diff
@@ -21,13 +24,286 @@ See the [official documentation](https://opensearch.org/docs/latest/) on how to
+
+#### Main steps
+
+These are the main steps of the setup:
+
+1. Install OpenSearch as detailed under [Install OpenSearch from an APT repository](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/debian/#install-opensearch-from-an-apt-repository).
+
+1. Follow [Step 2:(Optional) Test OpenSearch](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/debian/#step-2-optional-test-opensearch) from the installation guide. Remember to also **query the plugins endpoint**.
+
+1. Follow [Step 3: Set up OpenSearch in your environment](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/debian/#step-3-set-up-opensearch-in-your-environment) and change the *JVM Heap Space*.
+
+1. Set up a cluster as detailed under [Creating a cluster](https://opensearch.org/docs/latest/tuning-your-cluster/cluster/). Take the sections below into account when you do so.
+
+#### OpenSearch.yml configuration
+
+- For every node in your cluster, configure the *OpenSearch.yml* file as illustrated below (the example uses three nodes):
+
+  ```yml
+  # Use a descriptive name for your cluster:
+  #
+  cluster.name: NameOfYourCluster
+  cluster.initial_master_nodes: ["opensearchnode1"]
+  #
+  # ------------------------------------ Node ------------------------------------
+  #
+  # Use a descriptive name for the node(node.name can be anything you desire, but we recommend hostname of node):
+  #
+  # Make the node.name reflect the name of your node
+  node.name: opensearchnode1
+  #
+  # ----------------------------------- Paths ------------------------------------
+  #
+  # Path to directory where to store the data (separate multiple locations by comma):
+  #
+  path.data: /var/lib/opensearch
+  #
+  # Path to log files:
+  #
+  path.logs: /var/log/opensearch
+  # ---------------------------------- Network -----------------------------------
+  #
+  # Set the bind address to a specific IP (IPv4 or IPv6), best is to use the real-ip of the node:
+  #
+  network.host: ipAddressOfTheNode
+  
+  network.publish_host: ipAddressOfTheNode
+  #
+  # Set a custom port for HTTP:
+  #
+  http.port: 9200
+  #
+  # For more information, consult the network module documentation.
+  #
+  # --------------------------------- Discovery ----------------------------------
+  #
+  # Pass an initial list of hosts to perform discovery when this node is started:
+  # The default list of hosts is ["127.0.0.1", "[::1]"]
+   #
+  discovery.seed_hosts: ["ipAddressOfTheFirstNode","ipAddressOfTheSecondNode","ipAddressOfTheThirdNode"]
+  
+  discovery.type: zen
+  ```
+
+- If you want a node to be a **data node**, add the following configuration in *OpenSearch.yml*:
+
+  ```yml
+  node.roles: [ data, ingest ]
+  ```
+
+- If you want a node to be the **cluster manager node** (a.k.a. the master node), add the following configuration in *OpenSearch.yml*:
+
+  ```yml
+  node.roles: [ cluster_manager ]
+  ```
+
+#### TLS configuration
```

Comment:
Could you change this section about the TLS configuration to an actual procedure with numbered steps in chronological order? The way it's done now is quite confusing. As a user, I wouldn't know what to do when and where with this info right now. For example, when and where do I use the truststore files mentioned in the beginning?